helm/csi-powermax/values.yaml

```diff
@@ -53,7 +53,25 @@ version: "v2.7.0"
 
 images:
   # "driver" defines the container image, used for the driver container.
-  driverRepository: dellemc
+  driver: dellemc/csi-powermax:v2.7.0
+  csireverseproxy: dellemc/csipowermax-reverseproxy:v2.6.0
+  # CSI sidecars
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.2.2
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.8.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.9.0
+  # CSM sidecars
+  replication: dellemc/dell-csi-replicator:v1.5.0
+  vgsnapshotter: dellemc/csi-volumegroup-snapshotter:v1.2.0
+  podmon: dellemc/podmon:v1.6.0
```

Comment:
vgsnapshotter and podmon can be removed for powermax since support is not yet available.